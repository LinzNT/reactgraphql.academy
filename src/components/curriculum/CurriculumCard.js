import styled from 'styled-components'
import { RED, FONT_FAMILY } from '../../config/styles'

const CurriculumCard = styled.div`
  background-color: ${RED};
  padding: 2em;
  color: #ffffff;
  margin-top: 2em;
  ${FONT_FAMILY}
`

export default CurriculumCard
