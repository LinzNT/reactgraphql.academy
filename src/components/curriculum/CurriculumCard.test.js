import React from 'react'
import { shallow } from 'enzyme'
import { shallowToJson } from 'enzyme-to-json'

import CurriculumCard from './CurriculumCard'

describe('<CurriculumCard />', () => {
  it('should render component', () => {
    const wrapper = shallow(<CurriculumCard />)
    expect(shallowToJson(wrapper)).toMatchSnapshot()
  })
})
