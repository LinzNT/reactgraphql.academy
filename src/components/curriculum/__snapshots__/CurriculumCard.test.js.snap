// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<CurriculumCard /> should render component 1`] = `
<StyledComponent
  forwardedComponent={
    Object {
      "$$typeof": Symbol(react.forward_ref),
      "attrs": Array [],
      "componentStyle": ComponentStyle {
        "componentId": "sc-bdVaJa",
        "isStatic": true,
        "rules": Array [
          "
    background-color: ",
          "#be6045",
          ";
    padding: 2em;
    color: #ffffff
    margin-top: 2em;
    ",
          "font-family: 'Barlow', sans-serif;",
          "
",
        ],
      },
      "displayName": "styled.div",
      "foldedComponentIds": Array [],
      "render": [Function],
      "styledComponentId": "sc-bdVaJa",
      "target": "div",
      "toString": [Function],
      "warnTooManyClasses": [Function],
      "withComponent": [Function],
    }
  }
  forwardedRef={null}
/>
`;
